.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-.vscode
```

Comment:
we should only allow the settings.json and ignore everything else. see here https://github.com/shopware/shopware/blob/trunk/.gitignore#L20-L22

Reply:
resolved by https://github.com/shopware/docs/pull/1596/commits/1ac2aaf9885b8eb79932aeff5cf7bbb1d13566d6